gke/README.md

```diff
@@ -54,9 +54,41 @@ gcloud auth login
 
 # Deploy esp service
 ./setup_esp.sh
+```
 
+1. Copy the `config.yaml` file into the `/deploy/gke` folder. Rename
+the file to describe the environment the clusters are being used for.
+
+> The filename used will be the `<ENV>` in subsequent commands. E.g. if you
+> named the yaml file `staging.yaml`, then the `ENV` below is `staging`.
+
+1. Go to the Google Cloud Console and make sure the following APIs are enabled.
+   If the APIs are not yet enabled, enable them.
+   - Secret Manager API
+   - DataCommons API
+   - Generative Language API
+   - Maps Javascript API
+   - Places API
+
+1. In Google Cloud Console, under "APIs & Services" -> "Credentials", make sure
```

Comment:
Maybe add a todo to make this and next step in a shell script.

Reply:
Added the todo, will add script in followup PR.